### PR TITLE
Update README.md, highlighting "pip install minikey" as code, to make it more scannable

### PR DESCRIPTION
**Pull request by louislva**
My eyes scanned over it when I read it, so I thought I might save someone a minute or two

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,11 @@ board state and outputs an action.
 
 ## Installation
 
-pip install minihex
+Get it from PyPi:
 
-Editable installation (if you wish to tweak the environment):
+`pip install minihex`
+
+Or an editable installation (if you wish to tweak the environment):
```
Comment (FirefoxMetzger) [inline README.md]:
```suggestion
Or as an editable installation (if you wish to tweak the environment):
```

File: README.md [hunk 2]
```diff
@@ -71,4 +73,4 @@ Currently the enviornment is missing the following features to go to version 1.0
 - no surrender action
 
 ## Bugs and Contributing
-If you encounter problems, check the [GitHub issue page](https://github.com/FirefoxMetzger/minihex/issues) or open a new issue there.
\ No newline at end of file
+If you encounter problems, check the [GitHub issue page](https://github.com/FirefoxMetzger/minihex/issues) or open a new issue there.
```
Comment (FirefoxMetzger) [inline README.md:76]:
```suggestion
If you encounter problems, check the [GitHub issue page](https://github.com/FirefoxMetzger/minihex/issues) or open a new issue there.

```

**Review by FirefoxMetzger — APPROVED** [review]
Awesome stuff! Thanks for the PR.

There are two small tweaks I would do; I made them suggestions so you can co-author them if you wish (@louislva ). Otherwise, I will add them manually and merge the PR tomorrow.

**Comment by louislva** [comment]
Looks sweet! It's your project, do as you please.

**Pull request merged**